INSTALL.md

```diff
@@ -380,14 +383,33 @@ If you intend to run a server with translations, there are a few steps to follow
 
 1.  Prerequisites
 
-    Make sure gettext is installed (you need msgmerge and msgfmt, at least),
-    and the transifex client 'tx'
-    (http://help.transifex.com/features/client/index.html):
+    Make sure gettext is installed (you need msgmerge and msgfmt, at least):
+
+        sudo apt-get install gettext
 
-        sudo apt-get install gettext transifex-client
+    This will enable you to compile and install the translations that are in
+    the source repository.
+    
+    If you want to get the latest translation files or partial work-in-progress
+    translations, or wish to work on translations yourself, you will need to
+    create a [Transifex](https://www.transifex.com/) account and install its
+    client software (`tx`):
+
+        sudo apt-get install transifex-client
 
-    Configure a username and password in ~/.transifexrc using the format listed
-    on the above page.
+    More information (and alternative ways to install the client) can be found
+    at https://docs.transifex.com/client/introduction/.
+
+    Next, create an API token at https://www.transifex.com/user/settings/api/
+    and use it to configure your credentials in `~/.transifexrc`. Details at
+    https://docs.transifex.com/client/client-configuration#-transifexrc.
+    
+    Finally, you will need to join the
+    [MetaBrainz Foundation organization](https://www.transifex.com/musicbrainz/public/)
+    on Transifex to get access to the translations. If you wish to work on
+    translations, you will also need to join a language team at
+    (https://www.transifex.com/musicbrainz/musicbrainz/dashboard/); see also
+    https://musicbrainz.org/doc/Server_Internationalisation#Getting_started.
```

Comment:
Replacing the URLs with textual links might read a bit nicer in rendered form, e.g. `[join a language team](https://www.transifex.com/musicbrainz/musicbrainz/dashboard/)` instead of "join at language team at [long URL]," or `see also our documentation on [getting started with i18n](https://musicbrainz.org/doc/Server_Internationalisation#Getting_started).`

Reply:
Fair point, will change those too.